src/components/EventCalender/Yearly/YearlyEventCalender.spec.tsx

```
@@ -6,18 +6,41 @@ import {
-  SUPERADMIN = 'SUPERADMIN',
-  ADMIN = 'ADMIN',
-}
+import { BrowserRouter, MemoryRouter, useParams } from 'react-router-dom';
+import { UserRole } from 'types/Event/interface';
+
+// Mock the react-router-dom module
+vi.mock('react-router-dom', async () => {
+  const actual = await vi.importActual('react-router-dom');
+
+  // Create a proper React component that returns null
+  const MockNavigate = ({ to, replace }: { to: string; replace?: boolean }) =>
```

ESLint

[error] 'to' is defined but never used.

(@typescript-eslint/no-unused-vars)